Src/Localization/English/OpenShell.hhp

```diff
@@ -5,12 +5,67 @@ Contents file=OpenShellTOC.hhc
 Default topic=Main.html
 Display compile progress=Yes
 Language=0x409 English (United States)
-
+Title=Open-Shell Help
 
 [FILES]
 ClassicExplorer.html
 StartMenu.html
 ClassicIE.html
+ButtonTutorial.html
```

Comment:
> They are inextricably linked

No.
Commit `Change title of Help Window: Help -> Open-Shell Help` is supposed to just change the title of help window.
Adding some other things is certainly not related to help window title.
So those other things should go to other commit - `Add to Help two chapters` that adds those help chapters.

Reply:
Listen, what are we talking about?
I made a separate commit to change the title of Help Windows.
Only by changing .hpp files
Maybe I didn't understand you correctly?
![image](https://user-images.githubusercontent.com/25941601/215736335-9ed006cf-7a59-4f5b-9c19-5cecd38bddc3.png)

Reply:
> I made a separate commit to change the title of Help Windows.
> Only by changing .hpp files

That is right. But some of those `.hpp` files contain also other changes than just change in title.
Just have a look at diff of that commit:
https://github.com/Open-Shell/Open-Shell-Menu/pull/1317/commits/114369899ad9f3a789f5ab1d3e6d11041c3b0350

So you need to put those change so different commit (I guess they mixed in somehow).

Reply:
I named the commit "Add two chapters to help + Format header changes in all .html" because they are inextricably linked! And also all the changes are connected with the name of the PR - full localization.
It is absurd to create a commit after deleting each character I underline in the, not in the code, but in the documentation. By the way, I created a revert on two spaces in resource-file, as you asked.

Reply:
> I named the commit "Add two chapters to help + Format header changes in all .html" because they are inextricably linked!

And I'm talking about commit `Change title of Help Window: Help -> Open-Shell Help` (the one where I left this comment): 
https://github.com/Open-Shell/Open-Shell-Menu/pull/1317/commits/114369899ad9f3a789f5ab1d3e6d11041c3b0350

Just click on it, see the differences made in that commit and in `Src/Localization/English/OpenShell.hhp` and `Src/Localization/Russian/OpenShell.hhp` files you will see changes that are not related to change of help title.